app/router.js

```diff
@@ -63,7 +56,14 @@ Router.map(function () {
 
   this.route('sponsors');
 
-  this.route('survey');
+  this.route('survey', function () {
+    this.route('2016', { path: 'results-2016' });
+    this.route('2017', { path: 'results-2017' });
+    this.route('2018', { path: 'results-2018' });
+    this.route('2019', { path: 'results-2019' });
+    this.route('2020', { path: 'results-2020' });
+    this.route('2021', { path: 'results-2021' });
```

Comment:
I'd like to see the URLs in the format of `/survey/2021`, etc. so that it doesn't matter whether the page currently shows the results or the invitation to the survey. I think the shorter URL is also more memorable (e.g. presented in a slide more easily).

```suggestion
    this.route('2016', { path: '2016' });
    this.route('2017', { path: '2017' });
    this.route('2018', { path: '2018' });
    this.route('2019', { path: '2019' });
    this.route('2020', { path: '2020' });
    this.route('2021', { path: '2021' });
```